src/shaka_scheme/system/gc/GCList.cpp

```diff
@@ -4,67 +4,78 @@
 
 #include "shaka_scheme/system/gc/GCData.hpp"
 #include "shaka_scheme/system/gc/GCList.hpp"
+#include <utility>
 
 namespace shaka {
+    namespace gc {
 
-    GCList::GCList() {
-        this->size = 0;
-        this->head = nullptr;
-    }
+        GCList::GCList() {
```

Comment:
Use initialization list syntax if you can.

Reply:
Ok done!